tumblrWatch/com/5yp/tumblrWatch/finding_pharmercy.py

```diff
@@ -0,0 +1,24 @@
+import pytumblr
+
```

Comment:
It looks like this code repeats in a few places.
Consider moving the code to get the client connection to a separate file and import it.